# Movable/Ingredient
bacon: # scale 1
  grasp_pose: [0.0, 0.0, 0.02, 0.70710678, 0.70710678, 0.0, 0.0]
  grasp_size: 0.01
  place_orient: [0.0, 0.0, 0.0]

turkey: # scale 1
  grasp_pose: [0.0, 0.0, 0.01, 1.0, 0.0, 0.0, 0.0]
  grasp_size: 0.025
  place_orient: [0.0, 0.0, 0.0]

bread: # scale 1
  grasp_pose: [0.0, 0.0, 0.025, 0.0, 1.0, 0.0, 0.0]
  grasp_size: 0.01
  place_orient: [0.0, 0.0, 1.5707963]

garlic_bread: # scale 1
  grasp_pose: [0.0, 0.0, 0.025, 0.0, 1.0, 0.0, 0.0]
  grasp_size: 0.01
  place_orient: [0.0, 0.0, 1.5707963]

cheese: # scale 1
  grasp_pose: [0.0, 0.0, 0.01, 0.38268343, 0.92387953, 0.0, 0.0]
  grasp_size: 0.022
  place_orient: [0.0, 0.0, 0.0]

feta_cheese: # scale 1
  grasp_pose: [0.0, 0.0, 0.01, 0.38268343, 0.92387953, 0.0, 0.0]
  grasp_size: 0.022
  place_orient: [0.0, 0.0, 0.0]
  
ham: # scale 0.5
  grasp_pose: [0.0, 0.0, 0.01, 0.70710678, 0.70710678, 0.0, 0.0]
  grasp_size: 0.031
  place_orient: [0.0, 0.0, 0.0]
  
tomato: #scale 0.3
  grasp_pose: [-0.00029942837072416406, 0.0005543158215051136, 0.008728799140866772, 0.70457877370137645, 0.70859719652374098, -0.015044748591807686, 0.035105843786593045]
  # grasp_size: 0.04
  # grasp_size: 0.026
  grasp_size: 0.038
  place_orient: [0.0, 0.0, 0.0]

butter:
  grasp_pose: [0.0, 0.0, 0.012, 0.70710678, 0.70710678, 0.0, 0.0]
  grasp_size: 0.02
  place_orient: [0, 0, 0]

lettuce:
  grasp_pose: [0.0, 0.0, 0.012, 0.70710678, 0.70710678, 0.0, 0.0]
  grasp_size: 0.038
  place_orient: [0, 0, 0]

escarole:
  grasp_pose: [0.0, 0.0, 0.012, 0.70710678, 0.70710678, 0.0, 0.0]
  grasp_size: 0.038
  place_orient: [0, 0, 0]

dill:
  grasp_pose: [0.0, 0.0, 0.012, 0.70710678, 0.70710678, 0.0, 0.0]
  grasp_size: 0.038
  place_orient: [0, 0, 0]

mango: #scale 0.3
  grasp_pose: [-0.00029942837072416406, 0.0005543158215051136, 0.008728799140866772, 0.70457877370137645, 0.70859719652374098, -0.015044748591807686, 0.035105843786593045]
  # grasp_size: 0.04
  grasp_size: 0.038
  place_orient: [0.0, 0.0, 0.0]

avocado: #scale 0.3
  grasp_pose: [-0.00029942837072416406, 0.0005543158215051136, 0.008728799140866772, 0.70457877370137645, 0.70859719652374098, -0.015044748591807686, 0.035105843786593045]
  # grasp_size: 0.04
  grasp_size: 0.038
  place_orient: [0.0, 0.0, 0.0]

cucumber: #scale 1
  # grasp_pose: [-0.0032883911755783868, -0.57514355251661697, 0.020061915106673323, 0.96648932460240511, 0.0040132046490780303, 0.25652905863516512, 0.0086672771841701367]
  # grasp_pose: [0.0572, 0.0, 0.1, -0.9664403, 0.0, 0.2568915, 0.0]
  grasp_pose: [0.00858, 0.0, 0.015, -0.9664403, 0.0, 0.2568915, 0.0]
  grasp_size: 0.012
  # place_orient: [0.0193736, 0.5196087, -0.0051499]
  place_orient: [0.0, 0.5196087, 0.0]

sweet_pepper: #scale 1
  grasp_pose: [-0.0018956084753525282, -0.00067253516117882217, 0.021328269779857356, 0.70686108940441028, 0.70582309233071894, 0.042174378908403171, 0.019557207939835227]
  grasp_size: 0.025
  place_orient: [0, 0, 0]

green_pepper: #scale 1
  grasp_pose: [-0.0018956084753525282, -0.00067253516117882217, 0.021328269779857356, 0.70686108940441028, 0.70582309233071894, 0.042174378908403171, 0.019557207939835227]
  grasp_size: 0.025
  place_orient: [0, 0, 0]
  
onion: # scale 1
  grasp_pose: [0.0, 0.0, 0.02, 0.70710678, 0.70710678, 0.0, 0.0]
  grasp_size: 0.028
  place_orient: [0.0, 0.0, 0.0]

potato: # scale 1.3
  # grasp_pose: [-3.2950733411363231e-05, 0.00015542275766472372, 0.014932265830612856, 0.70700777594509967, 0.70720564261898555, 0.00027473373171955592, 0.00032912369761725976]
  # grasp_size: 0.02
  grasp_pose: [0.0, 0.0, 0.01, 0.0, -1.0, 0.0, 0.0]
  grasp_size: 0.015
  place_orient: [0, 0, 1.5707963]

banana: # scale 1
  # grasp_pose: [5.7731168567110025e-05, -0.34586235147650113, 0.012978589644869032, 0.99999990786739146, 0.00026175073031906027, 0.00018667279264869514, 0.000284438099199616]
  grasp_pose: [-0.0018022877548676819, 0.00015044346454518021, 0.016724028023826554, 0.99850009500189263, 0.00024739186077042051, 0.054641925322391975, 0.0034291508483173007]
  grasp_size: 0.018
  place_orient: [0, 0, 0]

black_olive: # scale 1
  grasp_pose: [1.6565308214588903e-05, -0.00039089115276794086, 0.01992701459674795, 0.70690860852961479, 0.70730475392096404, -0.00043328269025003844, 0.0001285831981105725]
  grasp_size: 0.01
  place_orient: [0, 0, 0]

walnut: # scale 1
  grasp_pose: [1.6565308214588903e-05, -0.00039089115276794086, 0.01992701459674795, 0.70690860852961479, 0.70730475392096404, -0.00043328269025003844, 0.0001285831981105725]
  grasp_size: 0.01
  place_orient: [0, 0, 0]

orange_pepper: # scale 1
  grasp_pose: [1.6565308214588903e-05, -0.00039089115276794086, 0.01992701459674795, 0.70690860852961479, 0.70730475392096404, -0.00043328269025003844, 0.0001285831981105725]
  grasp_size: 0.01
  place_orient: [0, 0, 0]

peanut: # scale 1
  grasp_pose: [1.6565308214588903e-05, -0.00039089115276794086, 0.01992701459674795, 0.70690860852961479, 0.70730475392096404, -0.00043328269025003844, 0.0001285831981105725]
  grasp_size: 0.01
  place_orient: [0, 0, 0]

shrimp: # scale 1
  grasp_pose: [-0.020006152687882972, 0.0020751157988525546, 0.012959181074606363, 0.70688386537914638, 0.70732956296554372, 0.000188201535872144, -0.0002340978249351622]
  grasp_size: 0.025
  place_orient: [0, 0, 0]

oregano: # scale 1
  grasp_pose: [1.6565308214588903e-05, -0.00039089115276794086, 0.01992701459674795, 0.70690860852961479, 0.70730475392096404, -0.00043328269025003844, 0.0001285831981105725]
  grasp_size: 0.01
  place_orient: [0, 0, 0]

cilantro: # scale 1
  grasp_pose: [1.6565308214588903e-05, -0.00039089115276794086, 0.01992701459674795, 0.70690860852961479, 0.70730475392096404, -0.00043328269025003844, 0.0001285831981105725]
  grasp_size: 0.01
  place_orient: [0, 0, 0]

cabbage: # scale 1
  grasp_pose: [1.6565308214588903e-05, -0.00039089115276794086, 0.01992701459674795, 0.70690860852961479, 0.70730475392096404, -0.00043328269025003844, 0.0001285831981105725]
  grasp_size: 0.01
  place_orient: [0, 0, 0]

scallion: # scale 1
  grasp_pose: [3.872168594709624e-05, -0.00041813958250805296, 0.014071064213204421, 0.70681576418214231, 0.7073974837824335, -0.00015823693997612877, 0.00050040280439985515]
  grasp_size: 0.013
  place_orient: [0, 0, 0]

celery: # scale 1
  grasp_pose: [3.872168594709624e-05, -0.00041813958250805296, 0.014071064213204421, 0.70681576418214231, 0.7073974837824335, -0.00015823693997612877, 0.00050040280439985515]
  grasp_size: 0.013
  place_orient: [0, 0, 0]

spinach: # scale 1
  grasp_pose: [3.872168594709624e-05, -0.00041813958250805296, 0.014071064213204421, 0.70681576418214231, 0.7073974837824335, -0.00015823693997612877, 0.00050040280439985515]
  grasp_size: 0.013
  place_orient: [0, 0, 0]

tuna:
  grasp_pose: [0.0, 0.0, 0.02, 0.70710678, 0.70710678, 0.0, 0.0] # test
  grasp_size: 0.035
  place_orient: [0, 0, 0]
  
# Movable/Ingredient Bottle
mayonnaise_bottle: # scale 0.5
  grasp_pose: [-0.018, 0.0, 0.0, 0.5, 0.5, 0.5, 0.5]
  grasp_size: 0.018
  place_orient: [0.0, 0.0, 0.0]

lemon_juice_bottle: # scale 0.5
  grasp_pose: [-0.018, 0.0, 0.0, 0.5, 0.5, 0.5, 0.5]
  grasp_size: 0.018
  place_orient: [0.0, 0.0, 0.0]

olive_oil_bottle: # scale 1 #0.003
  # grasp_pose: [-0.010013310092489403, 0.46534359021647304, 3.8180359875794601e-05, -0.49980162855855448, 0.4997565410580232, -0.50002351701856118, 0.50041803943136032]
  grasp_pose: [-0.003, 0.0, 0.0, -0.5, 0.5, -0.5, 0.5]
  grasp_size: 0.02
  place_orient: [0, 0, 0]

sugar_bottle: # scale 0.3
  grasp_pose: [0.0, 0.0, 0.0, 0.5, 0.5, 0.5, 0.5]
  grasp_size: 0.021
  place_orient: [0.0, 0.0, 0.0]
  
salt_bottle: # scale 0.3
  grasp_pose: [0.0, 0.0, 0.0, 0.5, 0.5, 0.5, 0.5]
  grasp_size: 0.021
  place_orient: [0.0, 0.0, 0.0]
  
pepper_bottle: # scale 0.3
  grasp_pose: [0.0, 0.0, 0.0, 0.5, 0.5, 0.5, 0.5]
  grasp_size: 0.021
  place_orient: [0.0, 0.0, 0.
  0]

parsley_bottle: # scale 0.3
  grasp_pose: [0.0, 0.0, 0.0, 0.5, 0.5, 0.5, 0.5]
  grasp_size: 0.021
  place_orient: [0.0, 0.0, 0.0]

relish_bottle: # scale 0.3
  grasp_pose: [0.0, 0.0, 0.0, 0.5, 0.5, 0.5, 0.5]
  grasp_size: 0.021
  place_orient: [0.0, 0.0, 0.0]

black_pepper_bottle: # scale 0.3
  grasp_pose: [0.0, 0.0, 0.0, 0.5, 0.5, 0.5, 0.5]
  grasp_size: 0.021
  place_orient: [0.0, 0.0, 0.0]

tuna_can:
  grasp_distance: 0.06
  grasp_pose: [-0.02, 0.0, 0.05, 0.50019299488910507, 0.50005945412085939, 0.49984951629812419, 0.49989796085275967]
  grasp_size: 0.03
  place_orient: [0, 0, 0]

# Movable/Tool
knife: # scale 1
  grasp_pose: [-0.01, 0.08, 0.0, 0.70710678, 0.0, 0.70710678, 0.0]
  grasp_size: 0.005
  place_orient: [0.0, 0.0, -1.5707963]
  place_pos: [0.0, -0.0482, 0.0]
  use_orient: [3.1415927, 0, 1.5707963]

spatula: # scale 1
  grasp_pose: [0.0, 0.12, 0.0, 0.70710678, 0.0, 0.70710678, 0.0]
  grasp_size: 0.002
  place_orient: [0.0, 0.0, -1.5707963]
  place_pos: [0.0, 0.0319, 0.0]
  use_orient: [-1.5707963, 0, -1.5707963]

scooper: # scale 1
  grasp_pose: [0.0, 0.0, 0.0, 1.0, 0.0, 0.0, 0.0]
  grasp_size: 0.0055
  # place_orient: [0.0, 0.0, -1.5707963]
  place_orient: [-1.5707963, 0.0, -1.5707963]
  place_pos: [0.0, -0.1037, 0.0]
  use_orient: [-2.75, 0.0, 1.5707963]

spreader: # scale 1
  grasp_pose: [0.0, 0.09, 0.0, 0.70710678, 0.0, 0.70710678, 0.0]
  grasp_size: 0.006
  place_orient: [0.0, 0.0, -1.5707963]
  place_pos: [0.0, -0.0509, 0.0]
  # use_orient: [0, 1.5707963, -1.5707963]
  use_orient: [3.1415927, 0, 1.5707963]
  
# Movable/Container
bowl: # scale 1
  grasp_pose: [0.0, -0.095, 0.08, 0.70710678, 0.70710678, 0.0, 0.0]
  grasp_size: 0.008
  place_orient: [0.0, 0.0, 0.0]
  puton_dist: 0.04

plate: # scale 1
  grasp_pose: [0.0, -0.095, 0.03, 0.70710678, 0.70710678, 0.0, 0.0]
  grasp_size: 0.015
  place_orient: [0.0, 0.0, 0.0]
  puton_dist: 0.01

pan: # scale 0.8
  grasp_pose: [-0.15, 0.0, 0.03625, 0.70710678, 0.70710678, 0.0, 0.0] # -0.03625
  grasp_size: 0.019
  place_orient: [0.0, 0.0, 0.0]
  puton_dist: 0.0

pot: # scale 0.7
  grasp_size: 0.01
  place_orient: [0.0, 0.0, 0.0]
  grasp_pose: [-0.099933074847225245, -0.19362854931443796, 0.11498795647205938, 1.0, 0.0, 0.0, 0.0]
  
# Fixed/Place
cutting_board: # scale 1
  place_orient: [0.0, 0.0, 0.0]
  
########################
# Chop/chopped object
chopped_onion:
  place_orient: [0, 0, 0]

chopped_cucumber:
  place_orient: [0, 0, 0]

chopped_sweet_pepper:
  place_orient: [0, 0, 0]

chopped_green_pepper:
  place_orient: [0, 0, 0]

chopped_feta_cheese:
  place_orient: [0, 0, 0]

chopped_cheese:
  place_orient: [0, 0, 0]

chopped_tomato:
  place_orient: [0, 0, 0]

chopped_potato:
  place_orient: [0, 0, 0]

chopped_cilantro:
  place_orient: [0, 0, 0]

chopped_oregano:
  place_orient: [0, 0, 0]

chopped_cabbage:
  place_orient: [0, 0, 0]

chopped_shrimp:
  place_orient: [0, 0, 0]

chopped_mango:
  place_orient: [0, 0, 0]

chopped_dill:
  place_orient: [0, 0, 0]

chopped_avocado:
  place_orient: [0, 0, 0]

chopped_scallion:
  place_orient: [0, 0, 0]

chopped_celery:
  place_orient: [0, 0, 0]

chopped_spinach:
  place_orient: [0, 0, 0]

chopped_banana:
  place_orient: [0, 0, 0]

chopped_tuna_sandwich:
  place_orient: [0, 0, 0]

chopped_tuna:
  place_orient: [0, 0, 0]

chopped_lettuce:
  grasp_pose: [0.0, 0.0, 0.012, 0.70710678, 0.70710678, 0.0, 0.0]
  grasp_size: 0.038
  place_orient: [0, 0, 0]

# Spread/spread object
spread_butter:
  place_orient: [0, 0, 0]

# Pour/bottled object 
mayonnaise:
  place_orient: [0, 0, 0]

salt:
  place_orient: [0, 0, 0]

sugar:
  place_orient: [0, 0, 0]

pepper:
  place_orient: [0, 0, 0]

parsley:
  place_orient: [0, 0, 0]

relish:
  place_orient: [0, 0, 0]

olive_oil:
  place_orient: [0, 0, 0]

lemon_juice:
  place_orient: [0, 0, 0]

black_pepper:
  place_orient: [0, 0, 0]

# tuna:
#   place_orient: [0, 0, 0]

# Stir/new object
tuna_spread:
  place_orient: [0, 0, 0]

# Finished object
tuna_sandwich:
  grasp_pose: [6.1901280107856671e-05, -3.320105119419825e-05, 0.030034954306542239, 0.70688864886428837, 0.70732460582292911, -0.00058166213772467275, -4.2111946146158799e-05]
  grasp_size: 0.01
  place_orient: [0, 0, -1.5707963]